On musl this is necessary to get full definition of pt_regs.

--- libraries/source/nvtt/src/src/nvcore/Debug.cpp
+++ libraries/source/nvtt/src/src/nvcore/Debug.cpp
@@ -27,6 +27,10 @@
 #	include <signal.h>
 #endif
 
+#if NV_OS_LINUX
+#	include <asm/ptrace.h> // struct pt_regs
+#endif
+
 #if NV_OS_LINUX || NV_OS_DARWIN || NV_OS_FREEBSD || NV_OS_OPENBSD
 #	include <unistd.h>	// getpid
 #endif
